driver/parser/check.go

```diff
@@ -466,29 +424,10 @@ func checkTimeInterval(start, end *float32, duration float32) error {
 }
 
 // checkValidatorConstraints makes sure that there is correct number of validators.
-// if during whole run there are just genesis validators, then one validator is enough
-// if there is creation of new validators trough sfc, then at all times there has to be at least two validators validating at every time
-// note during run validator won't be immediately registered for epoch, because it only happens during epoch seal,
-// therefore this check is not sufficient on its own
 func (s *Scenario) checkValidatorConstraints() error {
-
-	// count static validators within the node
-	gvCount, dynamicValidatorCount := s.GetStaticDynamicValidatorCount()
-	if s.NumValidators == nil {
-		s.NumValidators = &gvCount
-	}
-
-	// This must be check here for test to pass
-	// _must_ allow case where validator count = 0
-	if *s.NumValidators < 0 {
+	if s.NumValidators != nil && *s.NumValidators < 0 {
```

Comment:
```suggestion
	if s.NumValidators != nil && *s.NumValidators < 1 {
```

Also, the error message needs to be updated.